src/App.tsx

```diff
@@ -16,80 +17,125 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
+  NONE = 'none',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
 }
 
 type ReorderOptions = {
   sortType: SortType,
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType !== 'none') {
+    visibleGoods.sort((firstGood, secondGood) => {
+      switch (sortType) {
+        case 'alphabet':
+          return firstGood.localeCompare(secondGood);
+        case 'length':
+          return firstGood.length - secondGood.length;
+        default:
+          return 0;
+      }
```

Comment:
Kinda weird code. I would sort inside `switch-case`, removed condition `(sortType !== 'none')`  and add case 'none'. Also in default case, u should throw an error.

Reply:
Yeah, ty